examples-gallery/two-pendulum-cart-variable-gallery.py

```diff
@@ -0,0 +1,311 @@
+"""
+Upright a double pendulum.
+==========================
+
+A double pendulum is rotably attached to a cart which can move
+along the horizontal X axis. The goal is to get the double
+pendulum to an upright position in the shortest time possible,
+given an upper limit on the absolute value of the force that
+can be applied to the cart.
+Gravity points in the negative Y direction.
+
+**Constants**
+
+m1:     mass of the cart [kg]
+m2:     mass of the first pendulum [kg]
+m3:     mass of the second pendulum [kg]
+lx:     length of the first pendulum rods [m]
+iZZ1:   moment of inertia of the first pendulum about its end point [kg*m^2]
+iZZ2:   moment of inertia of the second pendulum about its end point [kg*m^2]
+g:      acceleration due to gravity [m/s^2]
```

Comment:
Please use RestructuredText math notation for these (`:math:\`m_1\``) or if you are showing Python variable names they should be enclosed in double backticks. Lastly this should be a RestructureText list to render properly. Here are the basics of RestructuredText: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
See how these render incorrectly if you don't use lists:
![image](https://github.com/csu-hmc/opty/assets/276007/47661548-c82d-42cf-8320-60d8db53dbcb)

Reply:
Would this be similar to latex / mardown cells in Jupyter notebooks?

Reply:
It is very similar to markdown but has slight different syntax. The same syntax is for lists. Search the internet for "RestructuredText".

Reply:
I tried to copy from your drone example. In a list, there should be a leading - ?

Reply:
HI Peter, please read the documents we share: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
OF COURSE I read them!
Understanding them is my issue! :-(
For example: In your programs I often find
# %%
# some text
I could not find the meaning of # %% in the sphinx documentation. I _assume_, it makes the some text stand out.

Reply:
To understand how to create this file you need these documents:

- https://sphinx-gallery.github.io/ : Explains how to format a py file as a mixture of RestructuredText formatting comments using `%%` as the primary marker.
- https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html : Gives the basics of RestructuredText formatting (similar to markdown syntax). This is the standard format for Python documentation.
- https://peps.python.org/pep-0008/ gives the style suggestions for the code itself (most projects try to follow most guidelines in this so that we don't have wildly different styles in code with multiple authors)